virt/virtual_machines/importing_vms/virt-importing-virtual-machine-images-datavolumes.adoc

```diff
@@ -7,6 +7,7 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 You can import an existing virtual machine image into your {product-title} cluster storage. Using the Containerized Data Importer (CDI), you can import the image into a persistent volume claim (PVC) by using a data volume. {VirtProductName} uses one or more data volumes to automate the data import and the creation of an underlying PVC. You can attach a data volume to a virtual machine for persistent storage.
+Use the Containerized Data Importer (CDI) to import a virtual machine image into a PVC by using a data volume. You can attach a PVC to a virtual machine for persistent storage.
```

Comment:
This sentence is duplicated from line 9 and makes things is a little unclear. This may be a little clearer for lines 9 & 10:

You can import an existing virtual machine image into your {product-title} cluster storage. Use the Containerized Data Importer (CDI) to import a virtual machine image into a persistent volume claim (PVC) by using a data volume. {VirtProductName} uses one or more data volumes to automate the data import and the creation of an underlying PVC. You can attach a PVC or a data volume to a virtual machine for persistent storage.

Reply:
Done; thanks